modules/utxo-lib/README.md

```diff
@@ -1,30 +1,8 @@
-# BitGo UTXO library (bitgo-utxo-lib)
-[![Build Status](https://travis-ci.org/BitGo/bitgo-utxo-lib.png?branch=master)](https://travis-ci.org/BitGo/bitgo-utxo-lib)
-[![NPM](https://img.shields.io/npm/v/bitgo-utxo-lib.svg)](https://www.npmjs.org/package/bitgo-utxo-lib)
-[![Known Vulnerabilities](https://snyk.io/test/github/BitGo/bitgo-utxo-lib/badge.svg?targetFile=package.json)](https://snyk.io/test/github/BitGo/bitgo-utxo-lib?targetFile=package.json)
+# BitGo UTXO library
 
-[![js-standard-style](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
-
-Originally a fork of [bitcoinjs-lib](https://github.com/BitGo/bitcoinjs-lib); we evolved this library to support the transaction building process of different UTXO based coins.
-
-## Supported coins
-- Bitcoin
-- Bitcoin Cash
-- Bitcoin Gold
-- Bitcoin SV (Satoshi Vision)
-- Dash
-- Litecoin
-- Zcash (Sapling compatible)
+This library is a wrapper around `bitcoinjs-lib` adding altcoin support.
```

Comment:
Having `bitcoinjs-lib` here link to the bitcoinjs-lib project would be nice, so people can quickly jump over there and find additional documentation/examples.